### DISCUSSION TITLE
When Cocoapods have latest version of Swift snapshot  library?

### DISCUSSION THREAD

**Discussion by AIengineerDev** [pointfreeco/swift-snapshot-testing#729] [General]
My projects use Cocoapods for dependencies. I notice that your library has the latest 1.9.0 for Cocopods, but latest version is 1.11.0.
And 1.9.0 release doesn't have iPhone 12 and 13 devices.  This is critical for many Swift projects.

**Comment by mbrandonw** [comment]
Hi @seniorQAautomationEngineer, we no longer support CocoaPods in this project, but you are free to maintain your own podspec so that you can continue using the library through CocoaPods.

**Reply by AIengineerDev** [reply]
Thanks a lot. I did. But the latest version in Cocoapods is only 1.9.0 with old devices.

**Reply by AIengineerDev** [reply]
Firewalls in my company block any Swift Package Manager dependencies. Do you have some workarounds for CocoaPods and a new version of your library? I have already many test cases. Honestly, I like your library and don't want to switch to another one.

**Reply by stephencelis** [reply]
Hi @seniorQAautomationEngineer, as @mbrandonw mentioned, you can always maintain your own custom podspec if you want to use a package that doesn't support CocoaPods.

Here's a discussion from another repo of ours where folks have defined their own custom podspec since we don't support CocoaPods: https://github.com/pointfreeco/swift-composable-architecture/issues/557#issuecomment-846223323

**Reply by AIengineerDev** [reply]
It helps. Thanks a lot.
